integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -241,3 +245,55 @@ def get_drafted_pages(
             "drafted_pages": drafted_pages,
             "single_drafted_page": single_drafted_page,
         }
+
+    @json_response
+    # pylint: disable=unused-argument, disable=no-self-argument
+    def get_translation_coverage_context(
+        request: HttpRequest, region_slug: str
+    ) -> JsonResponse:
+        r"""
+        Extend context by info on translation coverage of pages
+
+        :return: Dictionary containing the context for translation coverage of pages in a region
+        """
+        number_of_outdated_pages = 0
+
+        if not request.region.default_language:
+            return {}
+
+        languages = LanguageTreeNode.objects.filter(region=request.region).exclude(
+            language=request.region.default_language
+        )
+
+        pages_in_region = Page.objects.filter(
+            region=request.region, explicitly_archived=False
+        )
+        for page in pages_in_region:
+            if page.implicitly_archived:
+                pages_in_region.exclude(id=page.id)
+
+        possible_translations = languages.count() * pages_in_region.count()
+
+        published_foreign_translations = (
+            PageTranslation.objects.select_related("language")
+            .order_by("page__id", "language__id", "-version")
+            .distinct("page__id", "language__id")
+            .filter(page__region__slug=request.region.slug, minor_edit=False)
+            .exclude(language=request.region.default_language)
+            .all()
+        )
+
+        number_of_missing_translations = (
+            possible_translations - published_foreign_translations.count()
+        )
+
+        for pagetranslation in published_foreign_translations:
+            if pagetranslation.is_outdated:
+                number_of_outdated_pages += 1
+
+        return JsonResponse(
+            data={
+                "number_of_missing_or_outdated_translations": number_of_missing_translations
+                + number_of_outdated_pages
+            }
+        )
```

Comment:
I have tried to find a simplified calculation. It works, though the number is lower than what your calculation finds - also I am not sure it's actually faster. 



```suggestion
        non_archived_pages = request.region.non_archived_pages
        language_count = len(request.region.languages)
        possible_translations = non_archived_pages.count() * language_count

        all_translations = PageTranslation.objects.filter(
            page__in=non_archived_pages
        ).only("pk")
        non_outdated_translations = sum(
            1 for translation in all_translations if not translation.is_outdated
        )

        return JsonResponse(
            data={
                "number_of_missing_or_outdated_translations": possible_translations
                - non_outdated_translations
            }
        )
```

Reply:
Thank you a lot :pray: